src/type/definition.d.ts

```diff
@@ -271,6 +271,10 @@ export function getNamedType(type: GraphQLType): GraphQLNamedType;
  */
 export type Thunk<T> = (() => T) | T;
 
+export interface GraphQLScalarTypeExtensions {
```

Comment:
For a time being we need to maintain parity between TS and Flow so you should mirror these types into `definition.js`

Reply:
A type for Flow is 

```js
  extensions?: ?ReadOnlyObjMapLike<mixed>,
```

where

```js
export type ReadOnlyObjMap<T> = { +[key: string]: T, __proto__: null, ... };
export type ReadOnlyObjMapLike<T> =
  | ReadOnlyObjMap<T>
  | { +[key: string]: T, ... };
```

What's the resulting Flow type you'd like?

Maybe:

```js
export type GraphQLFieldConfigExtensions = ReadOnlyObjMapLike<mixed>;

// ...

  extensions: ?GraphQLFieldConfigExtensions;
```

Or maybe the more verbose:

```js
export type GraphQLFieldConfigExtensions =
  | {
      +[key: string]: mixed,
      __proto__: null,
      ... 
    }
  | {
      +[key: string]: mixed,
      ... 
    };

// ...

  extensions: ?GraphQLFieldConfigExtensions;
```

I assume the former.

Reply:
Done as the former.